packages/material-ui-lab/src/utils/clamp.js

```diff
@@ -0,0 +1,3 @@
+export default function clamp(value, min = 0, max = 100) {
```

Comment:
Can potentially be shared at some point with colorManipulator.

Reply:
That was the original intention - we have a clamp function in a couple of places. I plan to consolidate them once this is merged.